packages/kbn-aiops-utils/src/lib/fetch_stream.ts

```diff
@@ -41,7 +46,7 @@ export async function* streamFetch<E extends ApiEndpoint>(
 
     const bufferBounce = 100;
```

Comment:
Even though the code block below is pretty self-explainatory, I think it would be good to add a comment here explaining that we are buffering/batching the actions as an optimization. Not neccessarily but would be good to put this `100` into a constant as well for clarity?

Reply:
I moved `bufferBounce` to be an argument for the function so it can be custumized, also documented `fetchStream()` here: [304e8e3](https://github.com/elastic/kibana/pull/132590/commits/304e8e37ca577b281a22cd386f613e260a988f80).